app/test_main.py

```diff
@@ -1,3 +1,21 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expected",
+    [
+        pytest.param(0, 0, [0, 0]),
+        pytest.param(14, 14, [0, 0]),
+        pytest.param(15, 15, [1, 1]),
+        pytest.param(23, 23, [1, 1]),
+        pytest.param(24, 24, [2, 2]),
+        pytest.param(27, 27, [2, 2]),
+        pytest.param(28, 28, [3, 2]),
+        pytest.param(100, 100, [21, 17]),
+    ]
+)
+def test_get_human_age(cat_age: int, dog_age: int,
+                       expected: list[int]) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
```

Comment:
Ensure that the `get_human_age` function is correctly implemented to handle edge cases such as negative numbers or extremely large values, as suggested in the checklist. Although not explicitly required in the task description, it is a good practice to test these scenarios to ensure robustness.